sdk/src/main/java/com/google/cloud/dataflow/sdk/runners/DataflowPipelineTranslator.java

```diff
@@ -548,6 +551,7 @@ public void addStep(PTransform<?, ?> transform, String type) {
       currentStep.setKind(type);
       steps.add(currentStep);
       addInput(PropertyNames.USER_NAME, getFullName(transform));
+      addDisplayData(PropertyNames.DISPLAY_METADATA, DisplayData.from(transform));
```

Comment:
I'd propose we call the field DISPLAY_DATA (and put it in a similarly named location in the JSON). This way everything lines up.